Adding  a claim format version

edcdavid opened the pull request:
The claimFormat allows a client reading the claim file to check if it can decode it.

sebrandon1 reviewed (approved):
Did the schema.json changes happen automatically?

edcdavid commented:
> Did the schema.json changes happen automatically?

Do you mean schema.go? I modified schema.json and regenerated the schema.go (and updated the unit tests manually)
